### ISSUE TITLE
Windows errors on install of anaconda-client, anaconda-build

### ISSUE THREAD

**Issue by sdrobert** [ContinuumIO/anaconda-issues#7877]
Hi,

In addition to my failing Travis builds (#7876), my [Appveyor builds](https://ci.appveyor.com/project/sdrobert/pydrobert-kaldi) started failing. However, they are failing when installing anaconda-client and cond-build before the build actually starts:

```
conda install conda-build anaconda-client
Solving environment: ...working... done
## Package Plan ##
  environment location: C:\Miniconda3-x64
  added / updated specs: 
    - anaconda-client
    - conda-build
The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    pyyaml-3.12                |   py36h1d1928f_1         132 KB
    markupsafe-1.0             |   py36h0e26971_1          27 KB
    clyent-1.2.2               |   py36hb10d595_1          19 KB
    conda-build-3.2.0          |           py36_0         414 KB
    decorator-4.1.2            |   py36he63a57b_0          15 KB
    anaconda-client-1.6.6      |   py36ha174c20_0         193 KB
    beautifulsoup4-4.6.0       |   py36hd4cc5e8_1         133 KB
    python-dateutil-2.6.1      |   py36h509ddcb_1         238 KB
    ipython_genutils-0.2.0     |   py36h3c5d0ee_0          39 KB
    pkginfo-1.4.1              |   py36hb0f9cfa_1          56 KB
    traitlets-4.3.2            |   py36h096827d_0         131 KB
    jupyter_core-4.4.0         |   py36h56e9d50_0          84 KB
    glob2-0.6                  |   py36hdf76b57_0          17 KB
    jsonschema-2.6.0           |   py36h7636477_0         104 KB
    pytz-2017.3                |   py36h1d3fa6b_0         212 KB
    filelock-2.0.13            |   py36h20000bf_0          11 KB
    nbformat-4.4.0             |   py36h3a5bc1b_0         157 KB
    jinja2-2.10                |   py36h292fed1_0         184 KB
    conda-verify-2.0.0         |   py36h065de53_0          38 KB
    ------------------------------------------------------------
                                           Total:         2.2 MB
The following NEW packages will be INSTALLED:
    anaconda-client:  1.6.6-py36ha174c20_0 
    beautifulsoup4:   4.6.0-py36hd4cc5e8_1 
    clyent:           1.2.2-py36hb10d595_1 
    conda-build:      3.2.0-py36_0         
    conda-verify:     2.0.0-py36h065de53_0 
    decorator:        4.1.2-py36he63a57b_0 
    filelock:         2.0.13-py36h20000bf_0
    glob2:            0.6-py36hdf76b57_0   
    ipython_genutils: 0.2.0-py36h3c5d0ee_0 
    jinja2:           2.10-py36h292fed1_0  
    jsonschema:       2.6.0-py36h7636477_0 
    jupyter_core:     4.4.0-py36h56e9d50_0 
    markupsafe:       1.0-py36h0e26971_1   
    nbformat:         4.4.0-py36h3a5bc1b_0 
    pkginfo:          1.4.1-py36hb0f9cfa_1 
    python-dateutil:  2.6.1-py36h509ddcb_1 
    pytz:             2017.3-py36h1d3fa6b_0
    pyyaml:           3.12-py36h1d1928f_1  
    traitlets:        4.3.2-py36h096827d_0 
Downloading and Extracting Packages
pyyaml 3.12: ########## | 100% 
markupsafe 1.0: ########## | 100% 
clyent 1.2.2: ########## | 100% 
conda-build 3.2.0: ########## | 100% 
decorator 4.1.2: ########## | 100% 
anaconda-client 1.6.6: ########## | 100% 
beautifulsoup4 4.6.0: ########## | 100% 
python-dateutil 2.6.1: ########## | 100% 
ipython_genutils 0.2.0: ########## | 100% 
pkginfo 1.4.1: ########## | 100% 
traitlets 4.3.2: ########## | 100% 
jupyter_core 4.4.0: ########## | 100% 
glob2 0.6: ########## | 100% 
jsonschema 2.6.0: ########## | 100% 
pytz 2017.3: ########## | 100% 
filelock 2.0.13: ########## | 100% 
nbformat 4.4.0: ########## | 100% 
jinja2 2.10: ########## | 100% 
conda-verify 2.0.0: ########## | 100% 
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
reactivate does not accept arguments
remainder_args: ('conda-build', 'anaconda-client')
Command exited with code 1
```

I'm on conda 4.4.0, trying to install conda-build 3.2.0. This might be more general than trying to install either anaconda-client or anaconda-build, but I don't have a local machine to test with.

Sean

**Comment by msarahan** [comment]
Conda 4.4.0 was released today, and is likely the root of your problem.  @kalefranz please move this issue to the conda tracker.

**Comment by kalefranz** [comment]
Has been fixed in >4.4.0
